src/js/modules/plant/object.js

```diff
@@ -35,24 +32,28 @@ export default View.extend({
       options: this.app.data.options,
     });
 
-    this.$el.draggable({
-      cancel: '.icon-loop, .icon-trash, .icon-resize',
-    });
     this.model
       .on('change:currentProjection', this.updateProjection, this)
       .on('change:layerIndex', this.setLayer, this);
+
+    if (this.app.getState() !== Const.State.VIEWER) {
+      this.moveableElement();
+      this.on(MOVE_END, this.model.set, this.model);
```

Comment:
I actually thinking here about movint this listener definition into mixin module. We have access to `this` there and after that there will be no reason to export `MOVE_END` at all.